cmd/explaintest/t/ddl.test

```diff
@@ -0,0 +1,9 @@
+# issue 53176
```

Comment:
Remove integration test in release-6.5

Reply:
Why? It was in the tests/integrationtests, but I moved it to the explaintest to keep it covered also in release-6.5.
Or do you want me to convert it to a unit test (..._test.go)?